Marlin/temperature.cpp

```diff
@@ -1161,7 +1163,7 @@ void Temperature::disable_all_heaters() {
         PRR0
       #endif
         , PRSPI);
-    SPCR = _BV(MSTR) | _BV(SPE) | _BV(SPR0);
+    SPCR = _BV(MSTR) | _BV(SPE) | MAX6675_SPEED_BITS;
```

Comment:
Are we sure the sd-library initialises SPCR for every use (read/write) or does it make sense to store it here and restore later